### PR TITLE
Adds simple animated fabric activityIndicator

### PR DESCRIPTION
**Pull request by TatianaKapos**
## Description
Adds in a simple animated activityIndicator to Fabric. More work is needed to bring it to parity to the previous XAML ActivityIndicator such as adjusting the timing of the animation and size.

### Type of Change
- New feature (non-breaking change which adds functionality)

### Why
#11162

### What
Created an ActivityIndicatorVisual in CompostionContextHelper.cpp and set the visual in ActivityIndicatorComponentView

## Screenshots
Playground-composition_7vYOSzCPVy

## Testing
Tested Locally

### CODE REVIEW

File: vnext/Microsoft.ReactNative/CompositionSwitcher.idl [hunk 1]
```diff
@@ -128,7 +142,9 @@ namespace Microsoft.ReactNative.Composition
       Windows.Graphics.DirectX.DirectXAlphaMode alphaMode);
 
     SpriteVisual CreateSpriteVisual();
+    ShapeVisual CreateShapeVisual();
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/CompositionSwitcher.idl:145]: If you've got the CreateActivityVisual as part of this interface, then you probably do not need the CreateShapeVisual.  Your implementation of CreateActivityVisual can directly create wincomp shape visuals.

File: vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.cpp [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  if (!m_visual) {
+    m_visual = m_compContext.CreateSpriteVisual();
+    m_ActivityIndicatorVisual = m_compContext.CreateActivityVisual(); // creates COM control
+
+    OuterVisual().InsertAt(m_ActivityIndicatorVisual, 0);
+    OuterVisual().InsertAt(m_visual, 0);
+  }
+}
+
+void ActivityIndicatorComponentView::ensureDrawingSurface() noexcept {
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.cpp:102]: There is no custom rendering here, so you shouldn't use or create a drawingSurface

File: vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.cpp [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+      // Our update is done. EndDraw never indicates rendering device removed, so any
+      // failure here is unexpected and, therefore, fatal.
+      winrt::check_hresult(drawingSurfaceInterop->EndDraw());
+    }
+
+    // update color if needed
+    const auto activityIndicatorProps =
+        std::static_pointer_cast<const facebook::react::ActivityIndicatorViewProps>(m_props);
+    if (activityIndicatorProps->color) {
+      m_ActivityIndicatorVisual.updateColor(activityIndicatorProps->color.AsWindowsColor());
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.cpp:143]: Move this into updateProps

File: vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.h [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  void ensureVisual() noexcept;
+  void Draw() noexcept;
+  void ensureDrawingSurface() noexcept;
+
+  facebook::react::Size m_contentSize;
+  winrt::Microsoft::ReactNative::Composition::SpriteVisual m_visual{nullptr};
+  winrt::Microsoft::ReactNative::Composition::ActivityVisual m_ActivityIndicatorVisual{nullptr};
+  winrt::Microsoft::ReactNative::ReactContext m_context;
+  facebook::react::SharedViewProps m_props;
+  winrt::Microsoft::ReactNative::Composition::ICompositionDrawingSurface m_drawingSurface;
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.h:59]: remove

File: vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.h [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  ActivityIndicatorComponentView(
+      const winrt::Microsoft::ReactNative::Composition::ICompositionContext &compContext,
+      facebook::react::Tag tag,
+      winrt::Microsoft::ReactNative::ReactContext const &reactContext);
+
+  void ensureVisual() noexcept;
+  void Draw() noexcept;
+  void ensureDrawingSurface() noexcept;
+
+  facebook::react::Size m_contentSize;
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/Fabric/Composition/ActivityIndicatorComponentView.h:54]: remove

**Review by acoates-ms — CHANGES_REQUESTED** [review]

File: vnext/Microsoft.ReactNative/CompositionSwitcher.idl (hunk 1 above)
Comment (TatianaKapos) [reply vnext/Microsoft.ReactNative/CompositionSwitcher.idl]: good to know! Switched to use SpriteVisual instead

**Pull request merged**